gradle/libs.versions.toml

```diff
@@ -0,0 +1,231 @@
+facebook-react-android = { group = "com.facebook.react", name = "react-android", version.ref ="facebookReact" }
+facebook-react-hermesAndroid = { group = "com.facebook.react", name = "hermes-android", version.ref ="facebookReact" }
+facebook-shimmer = { group = "com.facebook.shimmer", name = "shimmer", version.ref ="facebookShimmer" }
+facebook-soLoader = { group = "com.facebook.soloader", name = "soloader", version.ref ="soLoader" }
+fastlane-screengrab = { group = "tools.fastlane", name = "screengrab", version.ref ="screengrab" }
+firebase-bom = { group = "com.google.firebase", name = "firebase-bom", version.ref = "firebaseBom"}
+firebase-iid = { group = "com.google.firebase", name = "firebase-iid", version.ref ="firebaseIid" }
+firebase-messaging = { module = "com.google.firebase:firebase-messaging" }
+glide = { group = "com.github.bumptech.glide", name = "glide", version.ref ="glide" }
+glide-ksp = { group = "com.github.bumptech.glide", name = "ksp", version.ref ="glide" }
+glide-volley-integration = { group = "com.github.bumptech.glide", name = "volley-integration", version.ref ="glide" }
+google-exoPlayer = { group = "com.google.android.exoplayer", name = "exoplayer", version.ref ="exoPlayer" }
+google-flexbox = { group = "com.google.android.flexbox", name = "flexbox", version.ref = "googleFlexbox" }
+google-gson = { group = "com.google.code.gson", name = "gson", version.ref = "gson" }
+google-material = { group = "com.google.android.material", name = "material", version.ref = "googleMaterial" }
+google-mlkit-barcode-scanning = { group = "com.google.mlkit", name = "barcode-scanning", version.ref ="googleMLKitBarcodeScanning" }
+google-mlkit-barcode-scanning-common = { group = "com.google.mlkit", name = "barcode-scanning-common", version.ref ="googleMLKitBarcodeScanningCommon" }
+google-mlkit-text-recognition = { group = "com.google.mlkit", name = "text-recognition", version.ref ="googleMLKitTextRecognition" }
+google-play-review = { group = "com.google.android.play", name = "review", version.ref ="googlePlayReview" }
+google-play-review-ktx = { group = "com.google.android.play", name = "review-ktx", version.ref ="googlePlayReview" }
+google-play-services-auth = { group = "com.google.android.gms", name = "play-services-auth", version.ref ="googlePlayServicesAuth" }
+greenrobot-eventBus = { group = "org.greenrobot", name = "eventbus", version.ref ="eventBus" }
+greenrobot-eventBus-java = { group = "org.greenrobot", name = "eventbus-java", version.ref ="eventBus" }
+indexos-media-for-mobile-android = { group = "com.github.indexos.media-for-mobile", name = "android", version.ref ="indexosMediaForMobile" }
+indexos-media-for-mobile-domain = { group = "com.github.indexos.media-for-mobile", name = "domain", version.ref ="indexosMediaForMobile" }
+installReferrer = { group = "com.android.installreferrer", name = "installreferrer", version.ref ="androidInstallReferrer" }
```

Comment:
Minor (🔍): Consider adding the `android` prefix to this dependency.

Reply:
Fixed.